src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -199,6 +203,18 @@ IntRect RenderTarget::getViewport(const View& view) const
 }
 
 
+////////////////////////////////////////////////////////////
+IntRect RenderTarget::getScissor(const View& view) const
+{
+    float            width   = static_cast<float>(getSize().x);
+    float            height  = static_cast<float>(getSize().y);
+    const FloatRect& scissor = view.getScissor();
+
+    return IntRect({static_cast<int>(0.5f + width * scissor.left), static_cast<int>(0.5f + height * scissor.top)},
+                   {static_cast<int>(0.5f + width * scissor.width), static_cast<int>(0.5f + height * scissor.height)});
```

Comment:
Is this "add half then cast to int" pattern a way of rounding to the nearest integer? Might I suggest `std::round` if that's our intention?